src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
 body {
+  margin: 0 auto;
+
+  max-width: 1200px;
+  min-width: 800px;
+  --blue-color: #00acdc;
+  --header-height: 23px;
```

Comment:
'--header-height'  is a bit confusing name for this variable
Consider renaming it - for instance `header-link-height`, `nav-element-height`, etc